resources/META-INF/studio-contribs.xml.template

```diff
@@ -0,0 +1,38 @@
+<!-- Defines Android Studio IDE-specific contributions and implementations. -->
```

Comment:
We may want to look into a system to generate plugin.xml and studio-contribs.xml from the templates and remove the others from source control. Or, have a task on travis fail if changes are made to one file but not to the the other (template and non-template), else they risk getting out of sync.

Reply:
If this sounds good, lets open an issue to track.

Reply:
We really need to get into the habit of using `plugin gen`.

Adding a fail-safe check to travis sounds good, too. See #1842.